homeassistant/components/frontend/__init__.py

```diff
@@ -357,8 +357,10 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     )
 
     local = hass.config.path("www")
-    if os.path.isdir(local):
-        hass.http.register_static_path("/local", local, not is_dev)
+    if not os.path.isdir(local):
+        os.mkdir(local)
+    hass.http.register_static_path("/local", local, not is_dev)
+    hass.http.register_static_path("/config/www", local, not is_dev)
```

Comment:
Its not just `<URL>/config/www`, but also `<URL>/www`.

I don't think we should add this. It feels wrong.

Reply:
Yeah I agree with Frenck. I don't like this either.